src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -185,6 +185,17 @@ internal NativeCommandProcessor(ApplicationInfo applicationInfo, ExecutionContex
 
             //Create input writer for providing input to the process.
             _inputWriter = new ProcessInputWriter(Command);
+
+            try
+            {
+                Host.BufferCell[,] bufferContents = this.Command.Context.EngineHostInterface.UI.RawUI.GetBufferContents(
+                    new Host.Rectangle(_startPosition, _startPosition));
+                _scrapeHostOutput = true;
```

Comment:
This can be made a static check so that we do it only once. And `_scrapeHostOutput` can be renamed to `s_supportScreenScrape` (or something similar).

Reply:
will change